core/src/main/java/com/klaytn/caver/methods/response/GovernanceStackingInfo.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.methods.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.klaytn.caver.utils.Utils;
+import org.web3j.protocol.core.Response;
+
+import java.math.BigInteger;
+import java.util.List;
+
+public class GovernanceStackingInfo extends Response<GovernanceStackingInfo.StakingInfo> {
```

Comment:
I think there is a typo.

Stacking should be Staking.
I checked this document. https://docs.klaytn.com/bapp/json-rpc/api-references/governance